src/structures/paging/mapper/mod.rs

```diff
@@ -82,13 +82,99 @@ pub trait Mapper<S: PageSize> {
     ///
     /// This function might need additional physical frames to create new page tables. These
     /// frames are allocated from the `allocator` argument. At most three frames are required.
+    ///
+    /// Parent page table entries are automatically updated with `PRESENT | WRITABLE | USER_ACCESSIBLE`
+    /// if present in the `PageTableFlags`.
```

Comment:
We should not here that, depending on the used mapper implementation, the `PRESENT` and `WRITABLE` flags might be set for parent tables even if they are not set for the mapping.